src/graticule.js

```diff
@@ -468,62 +468,25 @@ export function Graticule (paramMetricQHistoryReference, paramEle, ctx, offsetDi
     // showTimers();
   }
   this.parseStyleOptions = function (styleOptions, ctx) {
-    const parsedObj = {
-      skip: false,
-      connect: 3,
-      color: '#000000',
-      pointWidth: 2,
-      halfPointWidth: 1,
-      drawDots: false,
-      lineDash: [],
-      oddLineWidthAddition: 0
-    }
     if (styleOptions) {
-      const styleKeys = Object.keys(styleOptions)
-      // first parse Options for parsedObj
-      parsedObj.skip = !!styleOptions.skip
-      /* connect is responsible for the way the
-       * data points will be connected
-       * 1 = direct
-       * 2 = last
-       * 3 = next
-       */
-      switch (styleOptions.connect) { // Thomas: Magische Zahlen sind nicht schön
-        //         prinzipiell sind magische Zahlen nicht schön
-        // Inkonsistenz: Einmal Strings, einmal Zahlen
-        case 'next':
-          parsedObj.connect = 3
-          break
-        case 'last':
-          parsedObj.connect = 2
-          break
-        case 'direct':
-          parsedObj.connect = 1
-          break
-        case 'none':
-        default:
-          parsedObj.connect = 0
-          break
-      }
-      if (styleKeys.includes('width')) {
-        parsedObj.pointWidth = parseFloat(styleOptions.width)
-        parsedObj.halfPointWidth = Math.floor(styleOptions.width / 2.00)
+      if (styleOptions.width) {
```

Comment:
should use explicit comparison, probably `... !== undefined`.
(applies also to many of the following checks)